components/inputs/demo/input-color-palette.js

```diff
@@ -0,0 +1,67 @@
+import '../../button/button.js';
+import '../input-text.js';
+import { css, html, LitElement } from 'lit';
+import { ifDefined } from 'lit/directives/if-defined.js';
+
+class InputColorPalette extends LitElement {
```

Comment:
This is just a demo helper component that simulates what the MVC control will do by rendering stuff inside the `<slot>`:

<img width="372" alt="Screen Shot 2023-03-01 at 1 52 43 PM" src="https://user-images.githubusercontent.com/5491151/222236940-b52e206a-ef6b-46b2-aa53-45d37a32ecfa.png">